packages/security/src/SecurityService.ts

```diff
@@ -1,6 +1,15 @@
 import { SecurityCheck, SecurityCheckResult } from "./deviceTrust";
 
+/**
+ * Service module for handling performing and reporting checks.
```

Comment:
handling**,** performing and reporting possible security issues in mobile application. 

Worth to add info that separate plugin is required.